This tests fails in a constrained Poudriere env, passes after adjustments

--- cabal-install/cabal-install.cabal.orig	2026-09-30 10:37:26 UTC
+++ cabal-install/cabal-install.cabal
@@ -415,26 +415,26 @@ -- max path limit (still a problem for some ghc versio
 -- The test suite name must be keep short cause a longer one
 -- could make the build generating paths which exceeds the windows
 -- max path limit (still a problem for some ghc versions)
-test-suite mem-use-tests
-  import: warnings, base-dep, cabal-dep, cabal-syntax-dep, cabal-install-solver-dep
-  type: exitcode-stdio-1.0
-  main-is: MemoryUsageTests.hs
-  hs-source-dirs: tests
-  default-language: Haskell2010
-
-  ghc-options: -threaded -rtsopts "-with-rtsopts=-M16M -K1K"
-
-  other-modules:
-    UnitTests.Distribution.Solver.Modular.DSL
-    UnitTests.Distribution.Solver.Modular.DSL.TestCaseUtils
-    UnitTests.Distribution.Solver.Modular.MemoryUsage
-    UnitTests.Options
-
-  build-depends:
-    , cabal-install
-    , containers
-    , tasty >= 1.2.3 && <1.6
-    , tasty-hunit >= 0.10
+-- test-suite mem-use-tests
+--   import: warnings, base-dep, cabal-dep, cabal-syntax-dep, cabal-install-solver-dep
+--   type: exitcode-stdio-1.0
+--   main-is: MemoryUsageTests.hs
+--   hs-source-dirs: tests
+--   default-language: Haskell2010
+--
+--   ghc-options: -threaded -rtsopts "-with-rtsopts=-M16M -K1K"
+--
+--   other-modules:
+--     UnitTests.Distribution.Solver.Modular.DSL
+--     UnitTests.Distribution.Solver.Modular.DSL.TestCaseUtils
+--     UnitTests.Distribution.Solver.Modular.MemoryUsage
+--     UnitTests.Options
+--
+--   build-depends:
+--     , cabal-install
+--     , containers
+--     , tasty >= 1.2.3 && <1.6
+--     , tasty-hunit >= 0.10
 
 
 -- Integration tests that use the cabal-install code directly
